src/pbt/prophecy_build_tool.py

```diff
@@ -732,6 +732,7 @@ def test_python(self, path_pipeline_absolute, path_pipeline):
                         "--cov=.",  # generate coverage for module test
                         "--cov-report=xml",  # XML format
                         "--junitxml=report.xml",
+                        "--html=report.html --self-contained-html",
```

Comment:
can we add this on passing some argument like --report-type etc
otherwise please verify that it doesn't affect xml reports and generating both.